utils/docker/Dockerfile.ubuntu.20.04

```diff
@@ -72,23 +68,26 @@ RUN echo -e "<settings>\n\
 	</mirrors>\n\
 </settings>" > /root/.m2/settings.xml
 
-# Switch to new user
-#USER $USER
-WORKDIR /home/$USER
-
-# set NOBUILD to disable git clone & build
-ARG NOBUILD
-
-# Fetch DAOS code
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-  git clone --recurse-submodules https://github.com/daos-stack/daos.git; fi
-WORKDIR /home/$USER/daos
+# copy DAOS tree
+WORKDIR /home/daos
+COPY ./ .
 
 # Build DAOS & dependencies
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-  scons --build-deps=yes install PREFIX=/opt/daos; fi
+RUN scons --build-deps=yes -j 4 install PREFIX=/opt/daos
 
 # Set environment variables
 ENV LD_LIBRARY_PATH=/opt/daos/lib:/opt/daos/lib64:$LD_LIBRARY_PATH
 ENV PATH=/opt/daos/bin:$PATH
 ENV FI_SOCKETS_MAX_CONN_RETRY=1
+
+# Copy example config files to /opt/daos
+COPY utils/config/examples /opt/daos
+
+# Build java HDFS bindings
+# Currently disabled since it is failing
+RUN pushd src/client/java && \
+#    mvn clean install -T 1C -DskipITs -Ddaos.install.path=/opt/daos && \
```

Comment:
Ditto.  (Works here too according to https://github.com/daos-stack/daos/runs/1582021266?check_suite_focus=true.